pkg/helm/release/manager.go

```diff
@@ -275,9 +275,13 @@ func generatePatch(existing, expected runtime.Object) ([]byte, error) {
 	// fields added by Kubernetes or by the user after the existing release
 	// resource has been applied. The goal for this patch is to make sure that
 	// the fields managed by the Helm chart are applied.
+	// Add operations without a value (nulll) can be ignored
 	patchOps := make([]jsonpatch.JsonPatchOperation, 0)
+	removeOps := make([]jsonpatch.JsonPatchOperation, 0)
 	for _, op := range ops {
-		if op.Operation != "remove" {
+		if op.Operation == "remove" {
+			removeOps = append(removeOps, op)
+		} else if !(op.Operation == "add" && op.Value == nil) {
```

Comment:
Should not be just else?
What happens if it faces a conditional that is not defined here?

Reply:
I had issues with chart templates that contain fields with a value of `null`. During every reconciliation, the operator tried to apply these changes to Kubernetes, but Kubernetes ignores null values completely and they do not appear in for example `kubectl get -o yaml`. So one minute later, the operator tries it again and again..

Reply:
I will add a test case for this